include/libopencm3/stm32/g4/rcc.h

```diff
@@ -192,27 +234,30 @@
 #define RCC_PLLCFGR_PLLQ_SHIFT		21
 #define RCC_PLLCFGR_PLLQ_MASK		0x3
 
-#define RCC_PLLCFGR_PLLQEN		BIT20
+#define RCC_PLLCFGR_PLLQEN		(1 << 20)
 
 /* Division for PLLSAI3CLK, 0 == 7, 1 == 17 */
-#define RCC_PLLCFGR_PLLP		BIT17
+
 #define RCC_PLLCFGR_PLLP_DIV7		0
-#define RCC_PLLCFGR_PLLP_DIV17		RCC_PLLCFGR_PLLP
-#define RCC_PLLPEN			(1 << 16)
+#define RCC_PLLCFGR_PLLP_DIV17		1
+#define RCC_PLLCFGR_PLLP_SHIFT		17
+#define RCC_PLLCFGR_PLLP_MASK		0x1
+
+#define RCC_PLLCFGR_PLLPEN		(1 << 16)
 
 /** @defgroup rcc_pllcfgr_plln RCC_PLLCFGR PLLN values
- * Allowed values 8 <= n <= 127, VCO output between 64 and 344 MHz
+ * Allowed values 8 <= n <= 127, VCO output between 96 and 344 MHz
```

Comment:
I see where you got this from, but it's actually _also_ wrong, (in range 2, it's only allowed up to 128) and the refman in rev5 even dropped the numbers and just said, "within the range allowed in the datasheet" :)

Reply:
This should be fixed now.